module/VuFind/src/VuFind/Search/Factory/BlenderBackendFactory.php

```diff
@@ -108,7 +108,7 @@ public function __invoke(ContainerInterface $sm, $name, ?array $options = null)
         foreach (array_keys($backendConfig) as $backendId) {
             $backends[$backendId] = $backendManager->get($backendId);
         }
-        $blenderMappings = $yamlReader->get($this->mappingsConfig);
+        $blenderMappings = $yamlReader->get($this->mappingsConfig . '.yaml');
```

Comment:
Since we're likely to release this in 11.1, I think we need something like this to avoid breaking back-compatibility with custom factories (even though, in practice, I doubt it's likely to matter):

```suggestion
        // Legacy code may already include the '.yaml' extension; ignore it for safety:
        $blenderMappings = $yamlReader->get(str_ends_with($this->mappingsConfig, '.yaml') ? $this->mappingsConfig : $this->mappingsConfig . '.yaml');
```